### ISSUE TITLE
Poprawa frontu

### ISSUE THREAD

**Issue by kozlovvski** [azolkos/kalendarzsgh#6]
Siema!
Generalnie to nie mam za dużo do czynienia z pythonem, więc potrzebuję się upewnić, że dobrze coś kumam, zanim zacznę pisać.

Rozumiem, że kalendarz.py wystawia tablicę `data_array`, która później jest od razu dostępna globalnie w `templates/index.html`, tak?

**Comment by azolkos** [comment]
Tak, generalnie do widoku przekazywany jest numpy.array (na początku chciałem przekazywać cały dataframe, ale były jakieś problemy więc porzuciłem), który potem wystawiam w formie tabeli. Reszta działa tak, że czyta zaznaczenia i przekazuje z powrotem do backendu listę wybranych indeksów.
